rasa_sdk/endpoint.py

```diff
@@ -101,7 +101,7 @@ async def webhook(request: Request) -> HTTPResponse:
         try:
             result = await executor.run(action_call)
         except ActionExecutionRejection as e:
-            logger.error(e)
+            logger.debug(e)
```

Comment:
This one got changed to debug not because it contained PII (the only ones we raise contain only action names and slot names) but because it is expected behavior (the form rejects when you give it unexpected input). i clarified the response body message to reflect that, but that is why we are not raising an error.